src/halonium/service.nim

```diff
@@ -58,6 +58,11 @@ proc getDriverExe(kind: BrowserKind): string =
       NoSuchServiceExecutableException,
       "There is no service executable for Android. Please use a remote webdriver instead."
     )
+  if result.len == 0:
```

Comment:
You don't need to do this because case statements are exhaustive and a compiler error will be thrown if the case statements don't cover all BrowserKinds. That means this is essentially dead code since it will never be hit. Otherwise, PR looks good. Thanks for contributing!

Reply:
without this a error throw from webdriver side `'' could not be found in your PATH environment variable. Please see https://chromedriver.chromium.org/home [WebDriverException]`

so I just want avoid this unclear error message.

Reply:
Ah, I see. I misunderstood how findExe works. In that case, please do:

```nim
result = case kind
...
```

because in the case above, result is not actually set and the exception will always be thrown otherwise.